[FE-25] Create script to GET requester pays status of a workspace bucket

samanthahv opened the pull request:
https://broadworkbench.atlassian.net/browse/FE-25

Example output for disabled RP bucket:
![image](https://user-images.githubusercontent.com/68919432/235744285-2b61e94b-3a76-4ff1-a067-f05cff6dd421.png)

Example output for enabled RP bucket:
![image](https://user-images.githubusercontent.com/68919432/235745032-28866ee0-2e82-4e05-93cd-75def34b883b.png)

mmorgantaylor commented:
looks good! can you demonstrate some testing by running the script on both a RP and non-RP bucket and showing the script's output in the PR description?

samanthahv commented:
@mmorgantaylor i added a screenshot of the output for a non-RP bucket to the Jira ticket, but i can add it to this PR description along with a screenshot for a RP-enabled bucket

samanthahv commented:
@mmorgantaylor done!

mmorgantaylor reviewed (changes requested):
a couple questions/requests!

samanthahv commented:
addressed your comments in this commit [46cf4e3](https://github.com/broadinstitute/horsefish/pull/135/commits/46cf4e3a27f2840aa64d0e29840f56e87ebfa497)

mmorgantaylor commented on scripts/get_requester_pays_status.sh:
```diff
@@ -0,0 +1,67 @@
+if (( $# < 2 )); then
+  echo "Usage: $0 TERRA_PROJECT_ID PATH_TO_TERRA_BUCKET_PATH"
+  echo "The Terra bucket path can be formatted as gs://fc-XXXXX or fc-XXXXX"
+  echo "i.e you can run ./set_requester_pays.sh on project_id fc-12345"
```
change to `./get_requester_pays.sh`

mmorgantaylor replied:
also remove the word "on" in the command and add backticks - so I think this line should be 
```echo "i.e you can run `./get_requester_pays.sh project_id fc-12345`"```
